overlord/snapstate/snapmgr.go

```diff
@@ -102,6 +102,11 @@ func (ss *SnapSetup) TryMode() bool {
 	return ss.Flags&TryMode != 0
 }
 
+// RevertOp returns true if the operation is a revert
+func (ss *SnapSetup) RevertOp() bool {
```

Comment:
Ah, I see, thanks!

If it's just for doInstall, we probably don't need to persist it in SnapSetup. How about a `revert bool` argument for doInstall as an initial approach?